contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -224,14 +229,16 @@ contract GenesisProtocolLogic is IntVoteInterface {
         require(proposal.state == ProposalState.Boosted || proposal.state == ProposalState.QuietEndingPeriod,
         "proposal state in not Boosted nor QuietEndingPeriod");
         require(_execute(_proposalId), "proposal need to expire");
-        uint256 expirationCallBountyPercentage =
+        uint256 blocksSinceTimeOut =
```

Comment:
just use seconds, not blocks, as unit of measure (see above)

Reply:
it is blocks ,not very accurate calculation but good enough.
with the assumption that an average block time is 15 seconds.

Reply:
Does not seem so good to me. If the first block is mined after 14 seconds, then "blocksSinceTimeout = ((14).div(15) = 0.

Reply:
yes, the first step is 0.